src/FSharp.Stats/Matrix.fs

```diff
@@ -588,6 +610,20 @@ module Matrix = begin
     /// computes the row specific sample covariance matrix of a data matrix
     let rowSampleCovarianceMatrixOf (dataMatrix:Matrix<float>) =
         columnCovarianceMatrixOf (dataMatrix.Transpose.NumRows-1) dataMatrix.Transpose
+
+    /// computes the orientation and dataSource specific covariance matrix of a dataMatrix\
+    /// dataSource - "Sample" or "Population". \
+    /// orientation - "RowWise" or "ColWise" 
+    let covarianceMatrixOf (dataMatrix:matrix) (dataSource:DataSource) (orientation:Orientation) :matrix =
```

Comment:
Could you please change the parameter order, that dataMatrix is the last one. The same applies to sampleIntervalByZscore, populationIntervalByZScore, and mahalanobisDistanceList.